FlowCrypt/Functionality/DataManager/DataService.swift

```diff
@@ -210,6 +226,21 @@ extension DataService: ImapSessionProvider {
     }
 
     func smtpSession() -> SMTPSession? {
+        let net = SessionCredentialsService()
+        let email = "cryptup.tester@ukr.net"
+        let password = "HHjjdDVWqVZW96jP"
```

Comment:
This should be in secrets

Reply:
Sure! it's temporary password, not an account password